### ISSUE TITLE
This interaction failed

### ISSUE THREAD

**Issue by MagicSweet-dev** [cerus/jda-slash-commands#17]
I copypasted the given example and turned it into guild command, and:
![image](https://user-images.githubusercontent.com/49122140/120935595-37861080-c70c-11eb-8c30-686a9f24e6a7.png)

*whole code*
```java
	public static void main(String[] args) {
		loadJDA();
	}
	
	@SneakyThrows
	private static void loadJDA() {
		jda = JDABuilder.createDefault(
			configuration.tokens().token()
		).enableIntents(Arrays.asList(GatewayIntent.values())).setRawEventsEnabled(true).setMemberCachePolicy(MemberCachePolicy.ALL).setEventManager(new AnnotatedEventManager()).build().awaitReady();
		initCommands(jda);
	}
	
	public static void initCommands(JDA jda) {
		// Problem is not in 'configuration', i tried to hardcode the values, it also didn't work
		JDASlashCommands.initialize(jda, configuration.tokens().token(), configuration.tokens().applicationid());
		JDASlashCommands.submitGuildCommand(new CommandBuilder()
				.name("test-command") // Set command name to '/test-command'
				.desc("My cool test command")
				.option(new CommandBuilder.SubCommandGroupBuilder()
						.name("some-group") // Specify a group that can hold multiple sub commands
						.desc("This is a wonderful group")
						.option(new CommandBuilder.SubCommandBuilder()
								.name("hello") // Specify sub command 'hello' (/test-command some-group hello)
								.desc("Greet a user")
								.option(new ApplicationCommandOption(
										ApplicationCommandOptionType.USER,
										"user", // Note the lower case name - Names have to be lower case or else things could break
										"Specify a user to greet",
										true
								))
								.build())
						.option(new CommandBuilder.SubCommandBuilder()
								.name("animal") // Specify sub command 'animal' (/test-command some-group animal)
								.desc("Show a animal picture")
								.choices( // Only allow certain values: Cat, Dog and Platypus
										ApplicationCommandOptionType.STRING, // Specify type of the choice: STRING or INTEGER
										"animal", // Note the lower case name - Names have to be lower case or else things could break
										"Specify the animal",
										new ApplicationCommandOptionChoice("Cat", "cat"),
										new ApplicationCommandOptionChoice("Dog", "cat"),
										new ApplicationCommandOptionChoice("Platypus", "platypus")
								)
								.build())
						.build())
					// guild id
				.build(), 738377075387400233L, interaction -> {
					System.out.println("We got an interaction! Yay!");
					interaction.acknowledge();
				});
	}
```

I also tried:
- Run the bot on different machine, it could be my machine's or firewall problem or something
- Run the bot on different IP address (2 different addresses, actually)
- Make a simple slash command in Python (using [this library](https://github.com/discord-py-slash-commands/discord-py-slash-command)) to identify this is jda-slash-commands' issue and turns out it does work as intended *([image](https://cdn.discordapp.com/attachments/772253922118729748/851165729180745768/unknown.png))*, so this IS a jda-slash-commands' issue

What i also noticed - CommandInteractionEvent does not triggers in given example on ANY interaction *(on that Python-made one too)*:
```java	
private static class EventListen {
	@SubscribeEvent
	public void msg(CommandInteractionEvent event) {
		System.out.println(event);
	}
}
```

I don't know what else can I try, so I ask for some help here.
*(It would be easier to chat in Discord DM)*

**Comment by cerus** [comment]
Resolved in DMs

Solution: Don't use the `AnnotatedEventManager` until a workaround is found

**Comment by cerus** [comment]
Update: Fixed in 1.3.1
